test/functional/apps/management/_index_pattern_popularity.js

```diff
@@ -25,62 +25,61 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['settings', 'common']);
 
   describe('index result popularity', function describeIndexTests() {
```

Comment:
There are no tests in this describe block.

Reply:
addressed.